heron/packing/tests/java/org/apache/heron/packing/AssertPacking.java

```diff
@@ -42,7 +42,7 @@
  */
 public final class AssertPacking {
 
-  private static final double DELTA = 0.1;
+  public static final double DELTA = 0.1;
```

Comment:
same as #3187, is "public" necessary here?

Reply:
It's `public` in the test scope so it should be fine I think. I made it public because other test files involve comparing double too.